scrapy_po/utils.py

```diff
@@ -129,3 +133,26 @@ def build_instances(plan: andi.Plan, providers):
         else:
             instances[cls] = cls(**kwargs_spec.kwargs(instances))
     raise returnValue(instances)
+
+
+def callback_for(page_cls: Type[ItemPage]) -> Callable:
+    """Helper for creating callbacks for ItemPage sub-classes."""
+    if not issubclass(page_cls, ItemPage):
+        raise TypeError(
+            f'{page_cls.__name__} should be a sub-class of ItemPage.')
```

Comment:
I wonder if this check is needed - it would allow to fail slightly earlier, but previously there was no requirement to inherit from ItemPage, it was more like a helper, to ensure you don't forget to_item.